get_state_request.go

```diff
@@ -135,10 +135,15 @@ func (o *getStateOpts) buildPath() (string, error) {
 		channels = append(channels, utils.PamEncode(channel))
 	}
 
+	uuid := o.UUID
+	if uuid == "" {
```

Comment:
What happens if they don't use the `UUID()` at all in their builder? will it also default to `""`  and use the config UUID?

Reply:
then uuid is picked up from the config file.

Reply:
Perfect thanks!